src/Components/CookieConsent.js

```diff
@@ -0,0 +1,50 @@
+import * as React from "react";
+import * as Scrivito from "scrivito";
+import CookieConsent from "react-cookie-consent";
+
+/**
+ * @return {null}
+ */
+function CookieConsentBox() {
+  const root = Scrivito.Obj.root();
+  const cookieLink = root.get("cookiesPrivacyPolicyLink");
+  const cookiesLinkTitle = cookieLink && cookieLink.title();
+
+  if (!cookieLink) {
+    return null;
+  }
+
+  return (
+    <CookieConsent
+      buttonText="accept"
+      disableStyles="true"
```

Comment:
It should be `disableStyles={true}`. Otherwise you'll get a js console warning:

```
Warning: Failed prop type: Invalid prop `disableStyles` of type `string` supplied to `CookieConsent`, expected `boolean`
```